es-vue-base/src/lib-components/EsCtaBanner.vue

```diff
@@ -1,18 +1,23 @@
 <template>
     <es-card
-        :class="[{
-            'bg-gray text-white': dark,
-            'px-100 px-lg-200': variant === 'default',
-            'px-100 px-lg-300': variant === 'wide',
-        }]"
+        :class="[
+            {
+                'bg-gray text-white': dark,
+                'px-100 px-lg-200': variant === 'default',
+                'px-100 px-lg-300': variant === 'wide',
+            },
+        ]"
         v-bind="$attrs"
         v-on="$listeners">
         <b-row>
             <b-col
                 class="mb-200 my-lg-auto text-center text-lg-left"
                 :lg="lgFirst"
                 :xxl="xxlFirst">
-                <h2
+                <div
```

Comment:
Can we add a `<!-- -->` comment just above this, for future generations, that says something like "avoiding use of an `<h2>` tag here for long-form content SEO reasons, but preserving heading semantics for screen readers"? Just so no one thinks this is a bug later and changes it back.